docs/en/modules/modules-zephyr.adoc

```diff
@@ -25,6 +25,7 @@ roaster:
          foo: ABC-C2 #<10>
          bar: ABC-C3
       testCaseDepth: 3 #<11>
+      maxResults: 100 #<12>
```

Comment:
Ez így nem ad elég információt. Esetleg el lehetne gondolkozni az újrastrukturáláson:
```yml
roaster:
  zephyr:
    step:
      maxDepth: 3
      maxSize: 100
```

Reply:
A maxResults egy általánosabb paraméter a Zephyr apinál lekérdezéseknél, default 10-es értékkel. Lehetne külön hívásonként is, de azt már extrán túltolva érzem. Paraméterbe is csak azért raktam ki, hogy ne konstans legyen.

Reply:
Úgy látom, hogy csak a lépések lekérdezésénél van használva. A teljes zepyr-t nézve pedig a `maxResults` kifejezés nehezen értelmezhető.

Reply:
Berakva testSteps kulcs alá.